certification/internal/shell/openshift.go

```diff
@@ -0,0 +1,248 @@
+package shell
```

Comment:
Again, this one probably shouldn't be in shell either.

Reply:
Good point! I created a new package called k8s and moved the openshiftEngine and the check to this new package. Any feedback on the package name is welcome.